components/file_system/src/io_stats/biosnoop.rs

```diff
@@ -1,5 +1,7 @@
 // Copyright 2020 TiKV Project Authors. Licensed under Apache-2.0.
 
+//! This crate is deprecated.
```

Comment:
On the cloud env, we have strong control on the kernel. We can leverage it without root privilege. So let not mark it deprecated.